src/riscv-codegen.c

```diff
@@ -158,9 +158,11 @@ void emit_ph2_ir(ph2_ir_t *ph2_ir)
     int ofs;
 
     /* Prepare the variables to reuse the same code for
-     * the instruction sequence of division and modulo.
+     * the instruction sequence of
+     * 1. division and modulo.
+     * 2. load and store operations.
      */
-    rv_reg soft_div_rd = __t0, divisor_mask = __t1;
+    rv_reg helper_reg, divisor_mask = __t1;
```

Comment:
Instead of naming it helper_reg, how about `interm`, which is the abbreviated form of "intermediate"? i.e., place an intermediate register for the division/modulo and load-store instructions.